src/io_threads.c

```diff
@@ -297,6 +301,44 @@ void killIOThreads(void) {
     }
 }
 
+int updateIOThreads(const char **err) {
```

Comment:
When starting with a single thread, `initIOThreads` returns early without calling `prefetchCommandsBatchInit`, which we need to make sure we call it if we start with 1 then increase it

Reply:
I'll grab that as well